Resources/Prototypes/Recipes/Lathes/security.yml

```diff
@@ -463,8 +463,7 @@
     Steel: 500
     Glass: 100
     Plastic: 200
-
-- type: latheRecipe
+    
```

Comment:
You broke the uranium shell lathe recipe by accidentally deleting it's first line